sw/device/examples/sram_program/BUILD

```diff
@@ -128,12 +126,60 @@ opentitan_gdb_fpga_cw310_test(
         print sram_main()
 
         echo :::: Done.\\n
-    """,
-    gdb_script_symlinks = {
-        ":sram_program_fpga_cw310.elf": "sram_program.elf",
-        "//sw/device/silicon_creator/rom:rom_fpga_cw310.elf": "rom.elf",
-    },
-    rom_bitstream = "//hw/bitstream:rom_otp_exec_disabled",
-    rom_kind = "Rom",
-    tags = ["manual"],
+"""
+
+OTP_IMAGES = [
+    "test_unlocked0",
+    "dev",
+    "rma",
+]
+
+[
+    otp_image(
+        name = "img_{}_exec_disabled".format(otp_name),
+        src = "//hw/ip/otp_ctrl/data:otp_json_" + otp_name,
+        overlays = [
+            "//hw/ip/otp_ctrl/data:otp_json_creator_sw_cfg",
+            "//hw/ip/otp_ctrl/data:otp_json_owner_sw_cfg",
+            "//hw/ip/otp_ctrl/data:otp_json_hw_cfg",
+            "//hw/ip/otp_ctrl/data:otp_json_exec_disabled",
+        ],
+        visibility = ["//visibility:private"],
+    )
+    for otp_name in OTP_IMAGES
+]
+
+[
+    bitstream_splice(
+        name = "rom_otp_{}_exec_disabled".format(otp_name),
+        src = "//hw/bitstream:rom",
+        data = "img_{}_exec_disabled".format(otp_name),
+        meminfo = "//hw/bitstream:otp_mmi",
+        tags = ["manual"],
+        update_usr_access = True,
+        visibility = ["//visibility:private"],
+    )
+    for otp_name in OTP_IMAGES
+]
+
+[
+    opentitan_gdb_fpga_cw310_test(
+        name = "sram_program_fpga_cw310_test_otp_" + otp_name,
+        timeout = "short",
+        exit_success_pattern = "sram_program\\.c:\\d+\\] PC: 0x100020e0, SRAM: \\[0x10000000, 0x10020000\\)",
+        gdb_script = GDB_SCRIPT,
```

Comment:
are you guys going to check in the GDB script? :)
it will be interesting to see!

Reply:
The GDB script is already there! I moved into the GDB_SCRIPT variable in this PR, but kept its indentation the same, so it's kind of invisible in the diff.

Reply:
thanks for explaining in our chat Dan!